governance/votes/Executive Vote - September 27, 2024.md

```diff
@@ -0,0 +1,130 @@
+- **Proposal**: [Forum Post](https://forum.makerdao.com/t/smart-burn-engine-transaction-analysis-and-parameter-reconfiguration-update-9/25078)
+
+If this executive proposal passes, then the SBE Parameters will be updated through the following contract actions:
+
+#### Flapper Calls
+
+Initialisation of new Flapper by calling [FlapperInit.initFlapperUniV2](https://github.com/makerdao/dss-flappers/blob/master/deploy/FlapperInit.sol#L22) with the following parameters:
+
+- flapper_: FlapperUniV2SwapOnly ([0x374D9c3d5134052Bc558F432Afa1df6575f07407](https://etherscan.io/address/0x374D9c3d5134052Bc558F432Afa1df6575f07407))
+- want: 0.98 * WAD
+- pip: SWAP_ONLY_FLAP_SKY_ORACLE ([0x61A12E5b1d5E9CC1302a32f0df1B5451DE6AE437](https://etherscan.io/address/0x61A12E5b1d5E9CC1302a32f0df1B5451DE6AE437))
+- pair: PAIR_USDS_SKY ([0x2621CC0B3F3c079c1Db0E80794AA24976F0b9e3c](https://etherscan.io/address/0x2621CC0B3F3c079c1Db0E80794AA24976F0b9e3c))
+- usds: dss.chainlog.getAddress("USDS")
+- splitter: dss.chainlog.getAddress("MCD_SPLIT")
+- prevChainlogKey: bytes32(0)
+- chainlogKey: "MCD_FLAP"
+
+#### Oracle Wrapper
+
+Initialisation of new OracleWrapper by calling [FlapperInit.initOracleWrapper](https://github.com/makerdao/dss-flappers/blob/master/deploy/FlapperInit.sol#L38) with the following parameters:
+
+- wrapper_: SWAP_ONLY_FLAP_SKY_ORACLE ([0x61A12E5b1d5E9CC1302a32f0df1B5451DE6AE437](https://etherscan.io/address/0x61A12E5b1d5E9CC1302a32f0df1B5451DE6AE437))
+- divisor: 24,000
+- clKey: "FLAP_SKY_ORACLE"
+
+#### Additional Actions
+
+Additionally, the following parameters will updated by this spell:
+
+- Increase the [hop parameter (vow.hop)](https://sky-atlas.powerhouse.io/#A.3.5.1.1.3.1_Hop_Parameter-f9c3ba0f-3f7a-4222-9df8-efb0bc69433e%7C57ea2c549207d9fe7d45) by 1,386 seconds from 10,249 seconds to **11,635 seconds**.
+- Decrease the [bump parameter (vow.bump)](https://sky-atlas.powerhouse.io/#A.3.5.1.1.3.3_Bump_Parameter-512d164c-e60a-4054-850f-96285479901b%7C57ea2c549207d9fe7d45) by 40,000 USDS from 65,000 USDS to **25,000 USDS**.
+
+### Sky Ecosystem Liquidity Bootstrapping
+
+- **Authorization**: [Delegate Poll Request](https://forum.makerdao.com/t/atlas-edit-weekly-cycle-proposal-week-of-2024-09-23/25179/7), [Poll 1144](https://vote.makerdao.com/polling/QmfYzKTh)  
+- **Proposal**: [Forum Post](https://forum.makerdao.com/t/atlas-edit-weekly-cycle-proposal-week-of-2024-09-23/25179)
+
+If this executive proposal passes, then **10 million USDS** and **320 million SKY** will be transferred to Sky Ecosystem Liquidity Bootstrapping at [0xD8507ef0A59f37d15B5D7b630FA6EEa40CE4AFdD](https://etherscan.io/address/0xD8507ef0A59f37d15B5D7b630FA6EEa40CE4AFdD).
+
+### Aligned Delegate Compensation
+
+- **Authorization**: [Atlas A.1.5.5](https://sky-atlas.powerhouse.io/#A.1.5.5_Eligibility_To_Receive_Budget-5c9662be-9fc4-4b58-aef0-ca3e0bf56039|0db3af4ece0c)
```

Comment:
We had linked the old Atlas in the sheet since this is using the methodology described there